src/redux/slices/index.js

```diff
@@ -0,0 +1,8 @@
+import articleReducer from "./articles";
+import publisherReducer from "./publisher";
+
+const slicesReducer = {
+  news: articleReducer,
+  publushers: publisherReducer,
```

Comment:
You have a typo here. I guess you tried to say `publishers` not `publushers`. Could you recheck this and fix it?